titanic.ipynb

```diff
@@ -8,6 +8,7 @@
         "## Titanic 3\n",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 1 Line 3_
```markdown
## Titanic 3

This is a follow-up PR to test out how comments track changes over time
```
acooment

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/10)._